### ISSUE TITLE
Add a Competitive Programming Algorithm or Question.

### ISSUE THREAD

**Issue by kothariji** [kothariji/competitive-programming#1]
Step 1 - Fork this repository.
Step 2 - Clone the repository to yuor local machine.
Step 3 - Resolve this bugs, mentions provided in the Issues section of the repository. Also add a description what changes you ahve done.
Step 4 - Add the changes to your repository.
Step 5 - Create a PULL Request. And that's all.
NOTE - Please start the filename with platform name on which theproblem was solved. For eg if you are submitting graph problem from codechef then mention filename as (CODECHEF) chefandgraph.cpp.

**Comment by godslayer201** [comment]
Hey @kothariji , Can I add the long challenge solutions?

**Comment by kothariji** [comment]
> Hey @kothariji , Can I add the long challenge solutions?

@godslayer201 yeah sure! just add (CODECHEF) as a prefix to the file name and upload it to the particular folder like graphs or bits etc.

**Comment by godslayer201** [comment]
Added the same. 

Please review my PR and let me know if I should add anything. 

P.S : I will add the solutions,have just created the folder.

**Comment by kothariji** [comment]
> Added the same.
> 
> Please review my PR and let me know if I should add anything.
> 
> P.S : I will add the solutions,have just created the folder.

No No. actually we are arranging questions based on topics. So like you solved a question https://www.codechef.com/SEPT20B/problems/FINXOR of September Long challenge then add this question in bits manipulation folder with name *(CODECHEF)_Find_XOR.cpp*. No need to create a separate folder for codechef.  Any ways thank you for contributions

**Comment by chhipanikhil9** [comment]
can i add any topic ?

**Comment by kothariji** [comment]
> can I add any topic?

sure go ahead and add any topic relevant to competitive coding which is not previously formed!!!

**Comment by pradumio** [comment]
Your profile looks great with all kind of resources required for a competitive programmer. I would love to contribute. 
hey, @kothariji can I add the solution of project Euler problem 42 in CPP?

**Comment by kothariji** [comment]
> Your profile looks great with all kind of resources required for a competitive programmer. I would love to contribute.
> hey, @kothariji can I add the solution of project Euler problem 42 in CPP?

sure go ahead. Keep Contributing 🏆🥇🏆🥇

**Comment by pradumio** [comment]
Sure!

**Comment by godslayer201** [comment]
@kothariji Sure. I would do it that way and make the PR accordingly.

**Comment by chhipanikhil9** [comment]
why i unable to push ? it shows permission denied, sorry i am newbie on git

**Comment by kothariji** [comment]
> @kothariji Sure. I would do it that way and make the PR accordingly.

Great !

**Comment by kothariji** [comment]
> why i unable to push? it shows permission denied, sorry I am a newbie on  

@chhipanikhil9 fork the original repository and then clone your forked repo.

for more guidancence refer this - https://stackoverflow.com/questions/17857283/permission-denied-error-on-github-push

**Comment by chhipanikhil9** [comment]
> > why i unable to push? it shows permission denied, sorry I am a newbie on
> 
> @chhipanikhil9 fork the original repository and then clone your forked repo.
> 
> for more guidancence refer this - https://stackoverflow.com/questions/17857283/permission-denied-error-on-github-push

thanks 👍

**Comment by kothariji** [comment]
> > > why i unable to push? it shows permission denied, sorry I am a newbie on
> > 
> > 
> > @chhipanikhil9 fork the original repository and then clone your forked repo.
> > for more guidancence refer this - https://stackoverflow.com/questions/17857283/permission-denied-error-on-github-push
> 
> thanks for helping me in my first PR :)

Welocme to the world of OpenSource :confetti_ball::confetti_ball:

**Comment by OverthinkersArea** [comment]
@kothariji  Please review my PR.

**Comment by shreenanda-8** [comment]
Hey can I add solutions to A2oj ladder B?

**Comment by kothariji** [comment]
> Hey can I add solutions to A2oj ladder B?

Sure @shreenanda-8

**Comment by PrakharPipersania** [comment]
Hey, can I add Solution for Design Add & Search Words Data Structure (Trie)

**Comment by kothariji** [comment]
> Hey, can I add Solution for Design Add & Search Words Data Structure (Trie)

sure add a separate folder Trie and then push you code

**Comment by 27Anurag** [comment]
Hello, I'm a newbie on opensource and have a bit of experience in competitive coding. I would like to contribute to this repo, could you please assign me this issue. 
PS- would this contribution be counted towards hacktoberfest? 
Thanks!

**Comment by kothariji** [comment]
> Hello, I'm a newbie on opensource and have a bit of experience in competitive coding. I would like to contribute to this repo, could you please assign me this issue.
> PS- would this contribution be counted towards Oktoberfest?
> Thanks!

yes, go ahead.

**Comment by 27Anurag** [comment]
please check i've sent a pr

**Comment by kothariji** [comment]
> Hello, I've sent prs adding codes in some sections, could you please have a look.
> Thanks!
> And let me know if any changes are required, I'm a beginner.
sure!

**Comment by 27Anurag** [comment]
Hello, I saw you merged my previous pull request, thanks! 
I have sent another pr, please look into it,
Thanks!

**Comment by shreenanda-8** [comment]
Hey, in Codeforces, single question contains multiple tags.
Example: Greedy, math, sortings etc.
In this case, where should I add the solution?
(I mean, to which file should I add solution?)

**Comment by kothariji** [comment]
> Hey, in Codeforces, single question contains multiple tags.
> Example: Greedy, math, sortings etc.
> In this case, where should I add the solution?
> (I mean, to which file should I add solution?)

In that case you can choose the most suitable topic and can add into that.

**Comment by shreenanda-8** [comment]
> > Hey, in Codeforces, single question contains multiple tags.
> > Example: Greedy, math, sortings etc.
> > In this case, where should I add the solution?
> > (I mean, to which file should I add solution?)
> 
> In that case you can choose the most suitable topic and can add into that.

Thanks. 👍

**Comment by hrahul2605** [comment]
Hey @kothariji , Added a PR #46 , adds A2OJ problems ( Div2 B ). Please have a look at it.

**Comment by kothariji** [comment]
> Hey @kothariji , Added a PR #46 , adds A2OJ problems ( Div2 B ). Please have a look at it.

Done

**Comment by yogesh-kansal** [comment]
Can i add cp questions?

**Comment by yogesh-kansal** [comment]
@kothariji , Can i add some problems solutions related to competitive programming?

**Comment by kothariji** [comment]
> @kothariji , Can i add some problems solutions related to competitive programming?

Definitely

**Comment by Sweta040799** [comment]
Can I add problems with solutions of CodeChef's Cookoff and LunchTime problems

**Comment by kothariji** [comment]
> Can I add problems with solutions of CodeChef's Cookoff and LunchTime problems

@Sweta040799  yes you can, but do not make a separate folder for codechef,  push the lunchtime problems in there specific topic like in graph, bits etc. and don't forget to mention (CODECHEF) in starting of each file name.

**Comment by grvsood** [comment]
have a look #96

**Comment by nerdi-aayush** [comment]
Added solution to https://www.hackerrank.com/challenges/magic-square-forming/problem
script written in python 
pull request reference  #105

**Comment by grvsood** [comment]
@kothariji please add hacktoberfest-accepted label to the PR #96

**Comment by chhipanikhil9** [comment]
@kothariji please add "hacktoberfest-accepted" label to the all PR.

**Comment by kothariji** [comment]
> @kothariji please add "hacktoberfest-accepted" label to the PR #96



> @kothariji please add "hacktoberfest-accepted" label to the all PR.


@grvsood  @chhipanikhil9  done.!!!

**Comment by surya7765** [comment]
Hey, @kothariji I want to contribute to this repo.

**Comment by yk11-pro** [comment]
@kothariji i would like to contribute on this issue in c++,please assign it to me.

**Comment by kothariji** [comment]
> @kothariji i would like to contribute to this issue in c++, please assign it to me.

Sure go ahead.

**Comment by Sindhu-321** [comment]
I would like to add some c++ code. If the issue is open can I give it a try?

**Comment by kothariji** [comment]
> I would like to add some c++ code. If the issue is open can I give it a try?

@Sindhu-321 Sure go ahead.

**Comment by babitha667** [comment]
@kothariji can I add my Codeforces solutions.

**Comment by kothariji** [comment]
> @kothariji can I add my Codeforces solutions.

Sure go ahead.

**Comment by babitha667** [comment]
@kothariji  please review my pull request #203  . I have added my solutions of September codechef long challenge questions.

**Comment by kothariji** [comment]
> @kothariji please review my pull request #203 . I have added my solutions of September codechef long challenge questions.

👍

**Comment by aishwaryavasu0509** [comment]
@kothariji can I also add my CodeChef and code forces contest solutions in cpp ?

**Comment by kothariji** [comment]
> @kothariji can I also add my CodeChef and code forces contest solutions in cpp ?

yes you can, but make sure that questions should not be of live contest

**Comment by aishwaryavasu0509** [comment]
y

> > @kothariji can I also add my CodeChef and code forces contest solutions in cpp ?
> 
> yes you can, but make sure that questions should not be of live contest



> > @kothariji can I also add my CodeChef and code forces contest solutions in cpp ?
> 
> yes you can, but make sure that questions should not be of live contest

yeah sure

**Comment by Vivek-Hotti** [comment]
Hey there,
Please check my pull request #245 .

**Comment by kothariji** [comment]
> Hey there,
> Please check my pull request #245 .

Merged 🚀

**Comment by Vivek-Hotti** [comment]
Hey there,
Please check my pull request #246 .
By the way, wanted to say, this repository is amazing for newbies into competitive programming.

**Comment by kothariji** [comment]
> Hey there,
> Please check my pull request #246 .
> By the way, wanted to say, this repository is amazing for newbies into competitive programming.

Done. Merged 🚀 
Thank you so much.
Also thanks to you for the contributions.

Keep contributing 🙌 . 
Open Source is awesome❤️.

**Comment by Vivek-Hotti** [comment]
Please check my PR #247

**Comment by kothariji** [comment]
> Please check my PR #247

merged 🚀

**Comment by UG-SEP** [comment]
hey can i add [strong string](https://www.hackerrank.com/challenges/strong-password/problem) hackerrank string question 
can i add it in c language
please assign me

**Comment by kothariji** [comment]
> hey can i add [strong string](https://www.hackerrank.com/challenges/strong-password/problem) hackerrank string question
> can i add it in c language

Yes sure, please go forward. 📗📕 Refer CONTRIBUTING.md for contributions

**Comment by UG-SEP** [comment]
ok

**Comment by UG-SEP** [comment]
hey @kothariji can we add our project also like tic tac toe, Paint ,typing expert etc.
in c language

**Comment by kothariji** [comment]
> hey @kothariji can we add our project also like tic tac toe, Paint ,typing expert etc.
> in c language

Why not! But currently, we don't have a specific directory for adding projects, so just create a directory named *Projects*, and contribute your projects into that 🙌

**Comment by UG-SEP** [comment]
hey @kothariji please review my projects 
[Tic Tac Toe AI Intregrated](https://github.com/UG-SEP/Tic-Tac-Toe-AI-Intergrated)
[Paint With UG](https://github.com/UG-SEP/Paint-with-UG-The-SEP)
[UG Typing Expert](https://github.com/UG-SEP/UG-Typing-Expert)
if you like them then please give a star ⭐
Thanks

**Comment by kothariji** [comment]
> hey @kothariji please review my projects
> [Tic Tac Toe AI Intregrated](https://github.com/UG-SEP/Tic-Tac-Toe-AI-Intergrated)
> [Paint With UG](https://github.com/UG-SEP/Paint-with-UG-The-SEP)
> [UG Typing Expert](https://github.com/UG-SEP/UG-Typing-Expert)
> if you like them then please give a star ⭐
> Thanks

Damn 🔥 , Great work @UG-SEP 🚀 
I really liked - Paint-with-UG-The-SEP ⭐ 

Good Job 🙌

**Comment by UG-SEP** [comment]
> > hey @kothariji please review my projects
> > [Tic Tac Toe AI Intregrated](https://github.com/UG-SEP/Tic-Tac-Toe-AI-Intergrated)
> > [Paint With UG](https://github.com/UG-SEP/Paint-with-UG-The-SEP)
> > [UG Typing Expert](https://github.com/UG-SEP/UG-Typing-Expert)
> > if you like them then please give a star ⭐
> > Thanks
> 
> Damn 🔥 , Great work @UG-SEP 🚀
> I really liked - Paint-with-UG-The-SEP ⭐
> 
> Good Job 🙌

thanks a lot

**Comment by Amulya77** [comment]
hey @kothariji  can I Add CODECHEF lunchtime solutions of may.

**Comment by kothariji** [comment]
Yes you can @Amulya77  🙌

**Comment by navjotdadwal** [comment]
Sir please assign this issue to me

Why you should assign this to me?
i am current rated 5 star at codechef with rating of 2153 and rated expert at codeforces with rating of 1776
https://codeforces.com/profile/navjotdadwal
https://www.codechef.com/users/navjot1234
So i can handle and solve this issue

**Comment by kothariji** [comment]
> Sir please assign this issue to me
> 
> Why you should assign this to me? i am current rated 5 star at codechef with rating of 2153 and rated expert at codeforces with rating of 1776 https://codeforces.com/profile/navjotdadwal https://www.codechef.com/users/navjot1234 So i can handle and solve this issue

@navjotdadwal woah 🔥  your ratings looks so inspiring 🙌 
You can simply fork the repo, add your solutions and do a pull request 👍  I'll review it.

**Comment by navjotdadwal** [comment]
ok thanks sir

**Comment by avyayjain** [comment]
pls assign me this issue

**Comment by gaurangisinha** [comment]
Hi, @kothariji Could you please assign this issue to me too?

**Comment by kothariji** [comment]
@avyayjain  , @gaurangisinha  You can simply fork the repo, add your solutions and do a pull request 👍 I'll review it.

**Comment by 0901IT201046** [comment]
Sir i solved a codechef cookoff problem and ade a PR #305. Please reveiw and merge it.
Thankyou.

**Comment by Sun-0321** [comment]
how to figure out whether it is  hacktoberfest accepted or excluded project ?

**Comment by arsen1c** [comment]
Hey, I added Binary search (iterative and recursive approaches) in Javascript. @kothariji  Please check #385. Thanks :)

**Comment by JahnaviKashyap19** [comment]
Can I add codechef questions?

**Comment by arsen1c** [comment]
Added LinkedList in Javascript. Please check #397

**Comment by Anujkumar2017** [comment]
@kothariji 
Kindly review and merge PR #405

**Comment by SanjanaPurbia** [comment]
@kothariji  Can I work on this issue

**Comment by charchit-agarwal** [comment]
I am a beginner in open source contribution. I have added solution of codechef problem SUBTASK. Please accept my pull request.

**Comment by Hilbertmf** [comment]
Hello, I've added another Leetcode problem. This one involves dynamic programming. Please check #598

**Comment by Hilbertmf** [comment]
Hello, I've added a Codeforces problem. Please check #611

**Comment by gourangi-taware** [comment]
Can I add a Leetcode problem?

**Comment by kothariji** [comment]
> Can I add a Leetcode problem?

Yes @gourangi-taware , please go ahead 🚀

**Comment by manishtambe** [comment]
Can I add LinkedList problem from a Leetcode ?

**Comment by Sapna127** [comment]
Added the same.
Under the the Dynamic Programming folder.
Please review my PR and let me know if I should add anything.

**Comment by Kritika0804** [comment]
Added a Leetcode Probem. Please review my PR..

**Comment by Rakteem007** [comment]
can i contribute to this issue

**Comment by kunaldhongade** [comment]
please assign this issue to me!

**Comment by Vaibhavraj-nath-chauhan** [comment]
Please assign this issue to me

**Comment by SanjanaPurbia** [comment]
Please assign this issue to me

**Comment by vansh15** [comment]
Please assign this issue to me

**Comment by Nandinig24** [comment]
Please give issue to me

**Comment by urvashi-2004** [comment]
I'd like to do this. Please assign this issue to me.

**Comment by ankitkumar0622** [comment]
Hi, can you please assign me this task.

**Comment by xenom2004** [comment]
@kothariji please review my PR.

**Comment by Hilbertmf** [comment]
@kothariji review my Pull Request please
